data/avgi/avgi jobs.txt

```diff
@@ -0,0 +1,348 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+# Transport Jobs
+
+phrase "avgi specialists complete"
+	word
+		`You drop off the `
+	word
+		`engineers`
+		`doctors`
+		`scientists`
+		`researchers`
+		`mechanics`
+		`technicians`
+		`specialists`
+		`experts`
+		`academics`
+	word
+		` on <planet> as they chatter excitedly about their subjects of expertise. You collect <payment> in payment.`
+
+mission "Avgi: Specialized Workers 1"
+	job
+	repeat
+	name "Workers to <planet>"
+	description "Take these <bunks> to <planet> to provide rapid subject matter expertise in exchange for <payment>."
+	passengers 6 3 .45
+	to offer
+		random < 30
+	source
+		government "Avgi (Consonance)"
+	destination
+		government "Avgi (Consonance)" "Avgi (Dissonance)"
+	on visit
+		dialog phrase "avgi specialists complete"
+	on complete
+		payment 6000 300
+
+mission "Avgi: Specialized Workers 2"
+	job
+	repeat
+	name "Workers to <planet>"
+	description "Take these <bunks> to <planet> to provide rapid subject matter expertise in exchange for <payment>."
+	passengers 3 3 .3
+	to offer
+		random < 15
+	source
+		government "Avgi (Consonance)"
+	destination
+		government "Avgi (Consonance)" "Avgi (Dissonance)"
```

Comment:
```suggestion
	source
		government "Avgi (Consonance)"
	destination
		distance 1 100
		government "Avgi (Consonance)" "Avgi (Dissonance)"
```
You would proabably have to do something like the sol transport missions if you wanted to have intrasystem travel with this (and the other missions.)